Rakefile

```diff
@@ -556,14 +556,208 @@ task :update_licenses do
   load "util/generate_spdx_license_list.rb"
 end
 
+require_relative "bundler/spec/support/rubygems_ext"
+
+desc "Run specs"
+task :spec do
+  chdir("bundler") do
+    sh("bin/rspec")
+  end
+end
+
+namespace :dev do
+  desc "Ensure dev dependencies are installed"
+  task :deps do
+    Spec::Rubygems.dev_setup
+  end
+
+  desc "Ensure dev dependencies are installed, and make sure no lockfile changes are generated"
+  task :frozen_deps => :deps do
+    Spec::Rubygems.check_source_control_changes(
+      :success_message => "Development dependencies were installed and the lockfile is in sync",
+      :error_message => "Development dependencies were installed but the lockfile is out of sync. Commit the updated lockfile and try again"
+    )
+  end
+end
+
+namespace :spec do
+  desc "Ensure spec dependencies are installed"
+  task :deps => "dev:deps" do
+    Spec::Rubygems.install_test_deps
+  end
+
+  desc "Ensure spec dependencies for running in parallel are installed"
+  task :parallel_deps => "dev:deps" do
+    Spec::Rubygems.install_parallel_test_deps
+  end
+
+  desc "Run all specs"
+  task :all => %w[spec:regular spec:realworld]
+
+  desc "Run the regular spec suite"
+  task :regular do
+    chdir("bundler") do
+      sh("bin/parallel_rspec")
+    end
+  end
+
+  desc "Run the real-world spec suite"
+  task :realworld do
+    chdir("bundler") do
+      sh("BUNDLER_SPEC_PRE_RECORDED=1 bin/rspec --tag realworld")
+    end
+  end
+
+  namespace :realworld do
+    desc "Re-record cassettes for the realworld specs"
+    task :record do
+      chdir("bundler") do
+        sh("rm -rf spec/support/artifice/vcr_cassettes && bin/rspec --tag realworld")
+      end
+    end
+
+    task :check_unused_cassettes do
+      chdir("bundler") do
+        used_cassettes = Dir.glob("spec/support/artifice/used_vcr_cassettes/**/*.txt").flat_map {|f| File.readlines(f).map(&:strip) }
+        all_cassettes = Dir.glob("spec/support/artifice/vcr_cassettes/**/*").select {|f| File.file?(f) }
+        unused_cassettes = all_cassettes - used_cassettes
+
+        raise "The following cassettes are unused:\n#{unused_cassettes.join("\n")}\n" if unused_cassettes.any?
+
+        puts "No cassettes unused"
+      end
+    end
+  end
+end
+
+desc "Check RVM integration"
+task :check_rvm_integration do
+  # The rubygems-bundler gem is installed by RVM by default and it could easily
+  # break when we change bundler. Make sure that binstubs still run with it
+  # installed.
+  sh("RUBYOPT=-Ilib gem install rubygems-bundler rake && RUBYOPT=-Ilib rake -T")
```

Comment:
The second RUBYOPT should now be `-Ibundler/lib`, given before it was `-Ilib`.

Reply:
Ah, Thanks!